### DISCUSSION TITLE
[bug?] Native keyboard shortcuts are not working

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#16979] [General]
According to: https://yetiforce.com/en/knowledge-base/documentation/user-documentation/item/keyboard-shortcuts
keyboard shortcuts should be in place, but latest stable is not working with it. Badly navigation is needed due to removed navigation buttons in detail view. How to fix it and if fix is in place how to assign custom shortcuts to certain action?

thanks

**Comment by mariuszkrzaczkowski** [comment]
see new documentation
https://doc.yetiforce.com/user-guides/interface-guide/keyboard-shortcuts

if it doesn't work, create an issue with a full description
